assets/javascripts/kitten/components/box/link-box.js

```diff
@@ -1,53 +1,159 @@
 import React, { Component } from 'react'
-import classNames from 'classnames'
+import Radium, { StyleRoot } from 'radium'
+import PropTypes from 'prop-types'
 import { ArrowIcon } from 'kitten/components/icons/arrow-icon'
+import { Text } from 'kitten/components/typography/text'
+import { Marger } from 'kitten/components/layout/marger'
+import COLORS from 'kitten/constants/colors-config'
+import { ScreenConfig } from 'kitten/constants/screen-config'
+import { mediaQueries } from 'kitten/hoc/media-queries'
 
 export class LinkBox extends Component {
+  static propTypes = {
+    displayIcon: PropTypes.bool,
+    href: PropTypes.string,
+    isExternal: PropTypes.bool,
+    title: PropTypes.string.isRequired,
+    titeltag: PropTypes.string,
```

Comment:
La casse n'est pas la même que celle dans `defaultProps`. Ça devrait être :
```suggestion
    titelTag: PropTypes.string,
```

Reply:
`titelTag` comme on prononce ? (Désolé, j'ai vu la notif sur Slack, je m'en vais).

Reply:
Le :trollface: